components/zoho_meeting/actions/get-meetings/get-meetings.mjs

```diff
@@ -0,0 +1,42 @@
+import app from "../../zoho_meeting.app.mjs";
+
+export default {
+  name: "Get Meetings",
+  version: "0.0.1",
+  key: "zoho_meeting-get-meetings",
+  description: "Get a list of meetings. [See the documentation](https://www.zoho.com/meeting/api-integration/meeting-api/list-of-meeting-api.html)",
+  type: "action",
+  props: {
+    app,
+    index: {
+      label: "Index",
+      description: "The index or position of first meeting to list",
+      type: "integer",
+      optional: true,
+      default: 0,
+    },
```

Comment:
We should handle pagination within the action. `index` can be removed as a prop, and `listtype` should be added as a prop with the following choices: (all | past | week | month | today | upcoming) and with "all" as the default. The action should paginate through the meetings and return a maximum of `count` meetings.